tests/dataset/test_hbg_dataset.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+import pytest
+import os
+
+import pynwb
+from ipfx.stimulus import StimulusOntology
+from ipfx.dataset.hbg_nwb_data import HBGNWB2Data
+import allensdk.core.json_utilities as ju
+from dictdiffer import diff
+from tests.dataset.test_ephys_nwb_data import nwbfile_to_test
+
+@pytest.fixture
+def tmp_nwb_path(tmpdir_factory):
+    nwb = tmpdir_factory.mktemp("test_nwb_data").join("test_hbg_data.nwb")
+    return str(nwb)
+
+@pytest.fixture
+def hbg_nwb_data(tmp_nwb_path):
+
+    nwbfile = nwbfile_to_test()
+    print(tmp_nwb_path)
+
+    with pynwb.NWBHDF5IO(path=tmp_nwb_path, mode="w") as writer:
+        writer.write(nwbfile)
+
+    ontology =  StimulusOntology([[('name', 'ramp stimulus'), ('code', 'RAMP1')],
+                             [('name', 'extpinbath stimulus'), ('code', 'extpinbath')],
+                             [('name', 'extpbreakn stimulus'), ('code', 'extpbreakn')],
+                             [('name', 'Long square stimulus'), ('code', 'Long square')],
+                             [('name', 'Short square stimulus'), ('code', 'Short square')],
+                             [('name', 'Rheobase stimulus'), ('code', 'Rheobase')],
+                             [('name', 'Ramp stimulus'), ('code', 'Ramp')],
+                             [('name', 'Capacitance stimulus'), ('code', 'Capacitance')],
+                             [('name', 'Chirp stimulus'), ('code', 'Chirp')],
+                             [('name', 'extpexpend stimulus'), ('code', 'extpexpend')]
+                             ])
+
+    return HBGNWB2Data(nwb_file=tmp_nwb_path, ontology=ontology)
+
+def test_create_hbg(hbg_nwb_data):
+    assert isinstance(hbg_nwb_data,HBGNWB2Data)
+
+def test_main_dat(hbg_nwb_data):
```

Comment:
here you are testing a particular function `get_sweep_record` thus the name of the test should be `test_get_sweep_record`. Then you also need a test for get_stim_code_ext. Why are you calling it test_main_dat given that the test is using a fake nwb file?